common/src/main/java/io/honeycomb/opentelemetry/EnvironmentConfiguration.java

```diff
@@ -174,6 +186,14 @@ public static void enableOTLPTraces() {
         final String endpoint = getHoneycombTracesApiEndpoint();
         final String apiKey = getHoneycombTracesApiKey();
         final String dataset = getHoneycombTracesDataset();
+        final String serviceName = getServiceName();
+
+        // helpful to know if service name is missing
+        if (!isPresent(serviceName)) {
```

Comment:
It would be nice if the warns were combined with the setting of headers below so we're not checking things multiple times. See suggestion below.